specs/language/expressions.tex

```diff
@@ -230,6 +234,75 @@
 refers to an element index that is out of range of the vector type of the
 postfix expression before the dot.
 
+\Sub{Matrix Swizzle}{Expr.Post.Swizzle.Matrix}
+\begin{grammar}
+  \define{matrix-swizzle-component-sequence}\br
+  matrix-zero-indexed-swizzle-sequence\br
+  matrix-one-indexed-swizzle-sequence\br
+
+  \define{matrix-zero-indexed-swizzle-sequence}\br
+  matrix-zero-indexed-swizzle\br
+  matrix-zero-indexed-swizzle-sequence matrix-zero-indexed-swizzle\br
+
+  \define{matrix-zero-indexed-swizzle}\br
+  \terminal{\_m} zero-index-value\br
+
+  \define{zero-index-value} \textnormal{one of}\br
+  \terminal{0 1 2 3 }\br
+
+  \define{matrix-one-indexed-swizzle-sequence}\br
+  matrix-one-indexed-swizzle\br
+  matrix-one-indexed-swizzle-sequence matrix-one-indexed-swizzle\br
```

Comment:
I'm not exactly sure the best way to rewrite this, but as written this would require  `_m1_m2` instead of `_m12` right?